integration-tests/cli/helpers/utils.js

```diff
@@ -20,7 +21,11 @@ export function clearData(apiConfig, entityName) {
     method: 'GET',
   }
   return client.process(request, payload => {
-    const results = payload.body.results
+    // Built-in states cannot be deleted
+    const results =
+      entityName === 'states'
+        ? filter(payload.body.results, { builtIn: false })
```

Comment:
Interesting picking `lodash.filter` for it instead of saying `payload.body.results.every(result => result.builtIn)` which is part of almost all platforms now. [ref](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/every)

Reply:
🙈